docs/python/debugging.md

```diff
@@ -550,14 +574,23 @@ If you want to run Flask's development server in development mode, use the follo
 
 <a name="debugger-not-working"></a>
 
+## Support for deprecated Python versions
+The Python Debugger extension supports all [actively supported versions](https://devguide.python.org/#status-of-python-branches) of Python.
+
+To debug Python projects that use deprecated versions of Python such as Python 2.7 and Python 3.6, you can install any of the "2023.1.XXX" versions of the Python Debugger extension. You can do so by opening the **Extensions** view (`kb(workbench.view.extensions)`) and search for `@installed python debugger`. You should see the Python Debugger extension listed in the results. Select the gear icon and then the **Install Another Version** option, and choose the latest version that starts with "2023.1".
```

Comment:
Nit but might cause some confusion - Since there isn't the option to install another version from the gear icon within the extensions page just the results panel gear, it might be helpful to provide a screenshot here highlighting the gear icon in the search view. Or, tell users they can also install a different version by clicking the down arrow on the uninstall button.

Reply:
I am also thinking it might be nice if we could point them to a version that would support some of the older versions maybe with a table so it is easier for them to find. What do you think?

Reply:
Love the idea. From our offline convo I'll also remove this section from these docs and add a note to go to the README for Python version combability info. And then add a table with the versions that support each version